### DISCUSSION TITLE
PIMCORE 11 : when i tried to preview PDFs some pdf showing "Preview not available: PDF is being scanned. This may take a while."

### DISCUSSION THREAD

**Discussion by sandyHM** [pimcore/pimcore#17228] [Q&A / Support] [answered]
when i tried to preview PDFs some pdf showing "Preview not available: PDF is being scanned. This may take a while.".

What I have done.
1. Done all changes mentioned in [Pimcore Bug ](https://github.com/pimcore/pimcore/issues/16318).
2. Command `bin/console messenger:consume pimcore_asset_update`  is properly running.

![pdf preview not showing](https://github.com/pimcore/pimcore/assets/107986459/88550f7d-2760-4877-8185-f7fb2c3e2c9c)

**Comment by idna001** [comment]
Hi, 
there is an [open PR](https://github.com/pimcore/pimcore/pull/16956) in the issue you mentioned. So there is no fix till now. 
Probably you have to wait for the next release.

**Comment by kingjia90** [comment, marked as answer]
Thank you for posting!

Currently for unsafe PDF, it would be like that.
This can be disabled (at own risks) by https://github.com/pimcore/pimcore/blob/757375677dc83a44c6c22f26d97452cc5cda5d7c/doc/04_Assets/README.md?plain=1#L31 while we are working on an alternative way to configure that it would be displaying an image preview of the PDF first page and let user click to open to improve the user experience there for potential unsafe pdfs.

**Reply by sandyHM** [reply]
Thanks for replying fast.
in which file i need to add this line of code

**Reply by kingjia90** [reply]
in an own config yaml file to override the default value

**Reply by sandyHM** [reply]
Thanks,
will try.

**Reply by sandyHM** [reply]
thanks again @kingjia90 , Its's working as expected.
